autotest/gdrivers/dted.py

```diff
@@ -286,6 +286,19 @@ def test_dted_15():
     gdal.SetConfigOption('GDAL_DTED_SINGLE_BLOCK', None)
     return ret
 
+
+def test_dted_16():
+
+    gdal.SetConfigOption('DTED_APPLY_PIXEL_IS_POINT', 'TRUE')
```

Comment:
```suggestion
    with gdaltest.config_option('DTED_APPLY_PIXEL_IS_POINT', 'TRUE'):
```
and move what follows in a scope